cmd/earthly/org_cmds.go

```diff
@@ -359,6 +359,11 @@ func (app *earthlyApp) actionOrgMemberList(cliCtx *cli.Context) error {
 		return err
 	}
 
+	if len(members) == 0 {
+		app.console.Printf("No members in %s", orgName)
```

Comment:
the downside with a change like this, is now it's more difficult to use pipes and commands like `xargs`.

Reply:
I thought about this as well but I think `app.console.Printf` will write to Stderr.

Reply:
oh nice! win-win :)